### PR TITLE
Update wg-policy README.md

### PR DESCRIPTION
**Pull request by rficcaglia**
cleanup project list - backlog guardian, add whitepaper link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,10 +25,14 @@ Once you've been a reviewer, you can request to become an approver by filing a P
...
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
 
-## Projects
+## Projects - also see [Project Board](https://github.com/kubernetes-sigs/wg-policy-prototypes/projects/1)
 
-* [Gaurdian](guardian/README.md)
 * [Policy Report CRD](policy-report/README.md)
+* [Kubernetes Policy Whitepaper](https://docs.google.com/document/d/1C_YHezrlVllq2gzFiwQhZ0dOAINYLOrHswbubpQLCmQ/edit#)
+
+### Backlog or Retired
```
Comment (JimBugwadia) [inline README.md:33]: I would suggest naming this as `Archived Projects` to make the status clear.

`Backlog` suggests pending tasks, waiting to be assigned and become active. 

`Retired` seems ok, but could be confused with `Graduated`.

**Review by JimBugwadia — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rficcaglia) [reply README.md:33]: well actually in the FV case Backlog is much more accurate.  Do we want to spit this into Retired and Backlog?

Comment (JimBugwadia) [reply README.md:33]: ...or keep backlog items in Git (using a label)  and add a link to that on the page?

Comment (rficcaglia) [reply README.md:33]: @JimBugwadia  my git fu is not up to snuff - I committed to my fork branch but I don't see it here?

**Review by JimBugwadia — APPROVED** [review]

**Pull request merged**